gapic-generator-java/src/main/java/com/google/api/generator/gapic/protowriter/Writer.java

```diff
@@ -69,15 +69,13 @@ public static CodeGeneratorResponse write(
       writeSamples(gapicClazz, getSamplePackage(gapicClazz), classPath, jos);
     }
 
+    // package info may come as null if we have no services, but we will exit
+    // this function early if so.
```

Comment:
qq, is this also still true? I don't see `writePackageInfo()` exiting early.

Reply:
Not true anymore either. I removed the comment.